cost-analyzer/templates/kubecost-metrics-service-template.yaml

```diff
@@ -0,0 +1,22 @@
+{{- if .Values.kubecostMetrics }}
+{{- if .Values.kubecostMetrics.emitter }}
+{{- if .Values.kubecostMetrics.emitter.enabled }}
+apiVersion: v1
+kind: Service
+metadata:
+  name: {{ template "kubecost.kubeMetricsName" . }}
+  annotations:
+    prometheus.io/scrape: "true"
+    prometheus.io/port: {{ (quote .Values.kubecostMetrics.emitter.port) | default (quote 9005) }}
```

Comment:
isn't the standard term that this is just a prom "exporter"? here's an interesting resource if you haven't seen: https://prometheus.io/docs/instrumenting/writing_exporters/